### PR TITLE
Developer notice

### PR DESCRIPTION
**Pull request by vbuterin**
Adds a notice for developers to set their expectations.

### CODE REVIEW

**Comment by JustinDrake** [comment]
> "Shard chains and crosslink data" section as reasonably close to the expected final form

FWIW I haven't reviewed that section.

**Comment by vbuterin** [comment]
> FWIW I haven't reviewed that section.

You *did* merge all the PRs that comprise it :)

Apologies if I misinterpreted; can adjust the wording to make it less committal.

**Comment by JustinDrake** [comment]
> You _did_ merge all the PRs that comprise it :)

Ha! That was to encourage having all the phase 1 PRs merged in (even in draft/incomplete form) so that I could print out all of phase 1, sit down, and carefully read everything.

> can adjust the wording to make it less committal

Sounds good. I'm obsessing over phase 0 details and behind on all phase 1 stuff 😭

File: specs/core/1_shard-data-chains.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 **NOTICE**: This document is a work-in-progress for researchers and implementers. It reflects recent spec changes and takes precedence over the [Python proof-of-concept implementation](https://github.com/ethereum/beacon_chain).
 
-At the current stage, development teams are encouraged to treat the "Shard chains and crosslink data" section as reasonably close to the expected final form and worth starting development if they have spare resources, but the "Updates to the beacon chain" section, while fundamentally feature-complete, is still expected to undergo very considerable revision and review.
+At the current stage, Phase 1 is, while fundamentally feature-complete, is still subject to change. Development teams with spare resources may consider starting on the "Shard chains and crosslink data" section; at least basic properties, such as the fact that a shard block can get created every slot and is dependent on both a parent block in the same shard and a beacon chain block at or before that same slot, are unlikely to change, though details are likely to undergo similar kinds of changes to what Phase 0 has undergone since the start of the year.
```
Comment (prestonvanloon) [inline specs/core/1_shard-data-chains.md]: s/Phase 1 is, while fundamentally feature-complete, is still/Phase 1, while fundamentally feature-complete, is still/

**Pull request merged**